java_fibonacci.java.java

```diff
@@ -0,0 +1,25 @@
+package milestone6;
+
+import java.util.Scanner;
+
+public class FibonacciSeries {
```

Comment:
Main.java:5: error: class FibonacciSeries is public, should be declared in a file named FibonacciSeries.java
public class FibonacciSeries {
       ^
1 error

Try to improve the code by fixing the errors.

Happy Coding!!
Happy Contributing!!